src/main/java/com/truelayer/java/entities/beneficiary/Beneficiary.java

```diff
@@ -1,4 +1,4 @@
-package com.truelayer.java.merchantaccounts.entities.transactions.beneficiary;
+package com.truelayer.java.entities.beneficiary;
```

Comment:
is this move required ? I thought we were going to use a new beneficiary object for payouts.

What I mean is: 
- If they're different and this is expected, I'd keep this class (and its subclassen) in the transactions package. And bump the library to 6.4.0 (unless there are other breaking changes which I'm currently not seeing)
- if they're different but that's unexpected, i'm happy to DRY here and introduce the breaking change. And suggest the MAP team to adjust their docs if needed

Reply:
I might be missing your point here, sorry. Let me clarify better:
- payouts API has the concept of beneficiary in CreatePayout request, and in GetPayout response, and the structure is different between them
- beneficiary structure for GetPayout response is identical to the one already present in the transaction package, so I figured it was cleaner to move them to a "shared" package, instead of referencing merchant transactions package in payouts package
- beneficiary structure for CreatePayout request is different from the others so I created them in the payouts package

Does it make sense?

Reply:
Ah I see. yes, closing this.